fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogFragment.kt

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+import android.app.Dialog
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.navigation.fragment.findNavController
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.mozilla.fenix.R
+import org.mozilla.fenix.browser.BrowserFragmentDirections
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A bottom sheet fragment displaying the Firefox Translation Options dialog.
+ */
+class TranslationOptionsDialogFragment : BottomSheetDialogFragment() {
```

Comment:
One more question on layout, so in the Figma, in landscape, the panel is planned to span the full [width of the page](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7212%3A177225&mode=dev). However, this [design may change](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD/Full-Page-Translation?node-id=7335%3A209194&mode=dev).

I think if it is an easy fix, could you go ahead and make the panel full width? (If it isn't an easy fix, let's wait until we have the final design to make the change or follow-up on it.) What do you think?

Reply:
Done